Header header
int32 classification
float32 score